### PR TITLE
Development

### PR DESCRIPTION
**Pull request by cstich**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,47 @@
 pytzwhere [![Build Status](https://travis-ci.org/pegler/pytzwhere.svg)](https://travis-ci.org/pegler/pytzwhere) [![Coverage Status](https://coveralls.io/repos/pegler/pytzwhere/badge.svg)](https://coveralls.io/r/pegler/pytzwhere)
 =========
 
-pytzwhere is a Python library to lookup the timezone for a given lat/lng entirely offline
+This is the unstable branch. Use at your own discretion.
```
Comment (pegler) [inline README.md]: this should be changed

**Comment by cstich** [comment]
Yes. Just saw it too. Fixed it already.

**Comment by pegler** [comment]
great.  I'll push the new version to pypi when I get a minute

**Comment by cstich** [comment]
Cool :)

**Pull request merged**